pkg/cluster/task/init_config.go

```diff
@@ -45,6 +46,9 @@ func (c *InitConfig) Execute(ctx *Context) error {
 
 	err := c.instance.InitConfig(exec, c.clusterName, c.clusterVersion, c.deployUser, c.paths)
 	if err != nil {
+		if _, ok := err.(meta.ConfigCheckError); ok && c.ignoreCheck {
```

Comment:
It's better to use `errors.Cause` to extract the root cause of the returned error.